applications/asset_tracker/Kconfig

```diff
@@ -245,6 +245,17 @@ config CLOUD_BUTTON_INPUT
 	range 1 1 if BOARD_NRF9160_PCA20035NS
 	default 1
 
+config CLOUD_CONNECT_RETRY_DELAY
+	int "Cloud connection retry delay in seconds"
+	default 480
+	help
+	  Time to wait before trying to connect to the cloud again if the
+	  previous attempt failed (in seconds).
+
+config CLOUD_CONNECT_COUNT_MAX
+	int "Maximum number of cloud connection attempts"
+	default 8
+
```

Comment:
What happens if user set `CLOUD_CONNECT_RETRY_DELAY` to 30 and `CLOUD_WAIT_DURATION` to 60 and `CLOUD_CONNECT_COUNT_MAX` to 10 ?

I assume there are some inter dependencies between those number, and this should probably be stated in the help text for all 3 settings, so user can easily find them if when using `menuconfig`

Reply:
updated, hopefully it is more clear now.